src/main/java/io/ebean/dbmigration/migration/Column.java

```diff
@@ -39,13 +43,16 @@
  */
 @XmlAccessorType(XmlAccessType.FIELD)
 @XmlType(name = "", propOrder = {
-  "content"
+  "before", "after"
 })
 @XmlRootElement(name = "column")
 public class Column {
-
-  @XmlValue
-  protected String content;
```

Comment:
Hope content was not used in previous versions

Reply:
Yeah, I don't know what it is / was  - should be good removing